scripts/get_database_catalog.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
I always get empty result from executing this script on my dev pg, is it expected?

Reply:
Does your dev pg have any tables? Try running the generate dummy data script and then this?